README.md

```diff
@@ -46,7 +47,9 @@ To run this project you need to have:
 
 * Install the gems
 
-        $ bundle install
+        $ gem install bundler
+        $ gem install json
+        $ bundle install --deployment --without development test
```

Comment:
These instructions should be generic and not specific for production use.
You can add something like:
```
bundle install (if you are installing a production environment it's advisable to use --deployment --without development test
```